### PR TITLE
Add wraith emoji

### PR DESCRIPTION
**Pull request by twe4ked**
https://github.com/BBC-News/wraith

### CODE REVIEW

**Comment by orien** [comment]
It'd be great to list it on the readme also.

👍

File: img-buildkite-64.json [hunk 1]
```diff
@@ -840,5 +840,13 @@
     "aliases": [
       "quay.io"
     ]
+  },
+  {
+    "name": "wraith",
+    "image": "img-buildkite-64/wraith.png",
+    "category": "Buildkite",
+    "aliases": [
+      "wraith"
```
Comment (orien) [inline img-buildkite-64.json]: Do we need an alias of the same name?

Comment (toolmantim) [inline img-buildkite-64.json]: Nope, not needed :)

**Comment by toolmantim** [comment]
Great! This is good to merge without the alias, and added to the top of the readme (easiest way is to just add it manually)

**Comment by twe4ked** [comment]
Oops. Fixed.

**Comment by twe4ked** [comment]
Oops. Fixed.

**Comment by toolmantim** [comment]
Woot! Thanks @twe4ked 💖

**Comment by twe4ked** [comment]
Thanks :)

**Comment by toolmantim** [comment]
@twe4ked it's now live! ✨

**Pull request merged**